test/test_common/utility.h

```diff
@@ -515,6 +514,16 @@ class TestUtility {
                                                 ProtobufMessage::getStrictValidationVisitor());
   }
 
+  template <class MessageType> static void validate(const MessageType& message) {
```

Comment:
Why do you need this templated? Protobuf::Message& for first param is enough, no?

Reply:
@lizan good point. I will clean up in my next PR in this series, thanks.